xml/Currency.xml

```diff
@@ -915,12 +951,6 @@
                 <narrative>US Dollar (Next day)</narrative>
             </name>
         </codelist-item>
-        <codelist-item>
-            <code>USS</code>
-            <name>
-                <narrative>US Dollar (Same day)</narrative>
-            </name>
-        </codelist-item>
```

Comment:
Was this codelist item withdrawn?

Reply:
(since removing completely would constitute a breaking change.)

Reply:
Hi Andy, this code isn't withdrawn in the ISO codelist. No-one in IATI is using it, and as it is a non-complete codelist other codes from the relevant ISO code can be used. We have chosen not to replicate them all e.g. gold, silver, platinum

Reply:
I understand the others to be breaking changes because their codes have changed. And not this one?

Reply:
It’s marked as complete:
https://github.com/IATI/IATI-Codelists-NonEmbedded/blob/d1b4312c58346a87ec59b5220891cf418d978d05/xml/Currency.xml#L1

Additionally, nothing here suggests it’s only a partial list:
https://github.com/IATI/IATI-Codelists-NonEmbedded/blob/d1b4312c58346a87ec59b5220891cf418d978d05/xml/Currency.xml#L6-L9

In fact, I’m surprised to learn it’s only a partial list.

Reply:
Hmm, I'll check with the team on this one.

Reply:
> this code isn't withdrawn in the ISO codelist

Okay, but the intended purpose of the `withdrawn` attribute isn’t just to mirror the original codelist. It’s also there to add information that’s missing from the original codelist. The original proposal has more details: https://support.iatistandard.org/hc/en-us/articles/214390846

> No-one in IATI is using it

You mean data that’s in the IATI registry? That’s probably true, but tools like d-portal and OIPA and other portals and tools do pull in these codelists. So while it’s tempting to be pragmatic about it, it’s good to manage these codelists in a way that’s consistent with a documented process.

I think it would probably be okay to be more flexible with incomplete codelists. But again, I’d expect this to be documented on [the codelist management page](http://reference.iatistandard.org/203/codelists/codelist-management/).